### PR TITLE
Upgrade Ubuntu AMD images from 18.04 LTS to 22.04 LTS

### PR DESCRIPTION
**Pull request by ivdiazsa**
This PR is a helper to the same Ubuntu images upgrades taking place in the runtime repo: https://github.com/dotnet/runtime/pull/86194

The goal here is to change our Ubuntu AMD helix queues from 18.04 to 22.04. The motivation behind this is that 18.04 has transitioned from standard support to LTS support, and our VM's ought to run a standard support release.

### CODE REVIEW

**Comment by ivdiazsa** [comment]
@hoyosjs I also took the liberty of updating the docs I considered relevant to this change. If I missed any, I can add them subsequently. Same, if some or all are not yet ready to be updated, I can restore those changes.

**Comment by ivdiazsa** [comment]
@richlander Here's the arcade counterpart PR in progress FYI.

File: Documentation/Operations/Helix-Machine-Management/Helix-Machine-Lifecycle-Processes.md [hunk 1]
```diff
@@ -179,9 +179,9 @@ Machines in the following queues need to be updated to their latest patch versio
 Linux machines (raspbian 9 iot devices should NOT get updated)
 
 - alpine.amd64.tiger.perf
-- ubuntu.1804.amd64.owl.perf
-- ubuntu.1804.amd64.tiger.perf
-- ubuntu.1804.amd64.tiger.perf.open
+- ubuntu.2204.amd64.owl.perf
```
Comment (hoyosjs) [inline Documentation/Operations/Helix-Machine-Management/Helix-Machine-Lifecycle-Processes.md:182]: I can't comment about many of these ops docs. @ilyas1974 would probably be better suited as a reviewer here.

**Review by hoyosjs — APPROVED** [review]
For the yml, the changes are OK. I leave the md's to the folks from this repo.

**Comment by ivdiazsa** [comment]
Adding the `NO MERGE` tag because we need to figure out some stuff in the runtime repo before merging this one.

File: Documentation/Operations/Helix-Machine-Management/Helix-Machine-Lifecycle-Processes.md (hunk 1 above)
Comment (ilyas1974) [reply Documentation/Operations/Helix-Machine-Management/Helix-Machine-Lifecycle-Processes.md:182]: The Perf team is currently in the process of moving the on-prem systems to ubuntu 22.04. Right now, the only queue with systems in it is the ubuntu.2204.tiger.perf queue. The migration of the remaining queues is still in progress and is being tracked in https://github.com/dotnet/arcade/issues/13531

Comment (ivdiazsa) [reply Documentation/Operations/Helix-Machine-Management/Helix-Machine-Lifecycle-Processes.md:182]: Okok! Thanks for the update @ilyas1974. So, I took a look at the issue you linked, and just to ensure I understood well. Am I right in assuming then what this PR wants to accomplish would be also completed when #13531 is done?

Comment (ilyas1974) [reply Documentation/Operations/Helix-Machine-Management/Helix-Machine-Lifecycle-Processes.md:182]: Correct.

Comment (ivdiazsa) [reply Documentation/Operations/Helix-Machine-Management/Helix-Machine-Lifecycle-Processes.md:182]: Alright, thanks for the clarification. I will close this PR then.

**Comment by ivdiazsa** [comment]
Closing as per Ilya's explanation here: https://github.com/dotnet/arcade/pull/13954#discussion_r1296035636

**Pull request closed without merging**